lib/VerifiableCredentialStore.js

```diff
@@ -113,6 +113,38 @@ export default class VerifiableCredentialStore {
     return doc.content;
   }
 
+  /**
+   * Updates a verifiable credential in remote private storage
+   *
+   * @param {object} credential
+   * @param {object} meta
+   */
+  async update({credential, meta = {}}) {
+    try {
+      const {invocationSigner} = this;
+      const {documents: [doc]} = await this.edv.find({
+        equals: {'content.id': credential.id},
+        invocationSigner
+      });
+      if(!doc) {
+        return false;
```

Comment:
I see that this is returning `false` here and on L142 when the document is not found.  Why is the `NotFoundError` being concealed from the API consumer?  In other words, why not throw a useful error message here instead of returning `false`?